build/teamcity-testrace.sh

```diff
@@ -36,20 +31,29 @@ fi
 tc_end_block "Determine changed packages"
 
 tc_start_block "Compile C dependencies"
-run build/builder.sh make -Otarget c-deps GOFLAGS=-race
+# Buffer noisy output and only print it on failure.
+run build/builder.sh make -Otarget c-deps GOFLAGS=-race &> artifacts/race-c-build.log || (cat artifacts/race-c-build.log && false)
+rm artifacts/race-c-build.log
 tc_end_block "Compile C dependencies"
 
+tc_start_block "Maybe stressrace pull request"
+build/builder.sh go install ./pkg/cmd/github-pull-request-make
+build/builder.sh env COCKROACH_QUIET_START=true BUILD_VCS_NUMBER="$BUILD_VCS_NUMBER" TARGET=stressrace github-pull-request-make
+tc_end_block "Maybe stressrace pull request"
+
 tc_start_block "Run Go tests under race detector"
 true >artifacts/testrace.log
 for pkg in $pkgspec; do
 	run build/builder.sh env \
 		COCKROACH_LOGIC_TESTS_SKIP=true \
+		COCKROACH_QUIET_START=true \
 		stdbuf -oL -eL \
 		make testrace \
 		PKG="$pkg" \
 		TESTTIMEOUT=45m \
 		TESTFLAGS='-v' \
 		USE_ROCKSDB_ASSERTIONS=1 2>&1 \
+		| grep -av 'More existing levels in DB than needed' \
```

Comment:
It seems kind of brittle to just grep out these ones, doesn't it?

Reply:
yeah, this is temporary until if/when we remove that logging line from our fork of rocks -- I just didn't want to mess with forks/submodules right now.

Reply:
replaced by cockroachdb/go-test-teamcity#12 for now (though I think we want to comment it out for real for humans too at some point)